core/store/models/job_spec_test.go

```diff
@@ -46,8 +46,21 @@ func TestJobSpec_Save(t *testing.T) {
 	store, cleanup := cltest.NewStore(t)
 	defer cleanup()
 
+	befCreation := time.Now()
 	j1 := cltest.NewJobWithSchedule("* * * * 7")
+	aftCreation := time.Now()
+
+	assert.True(t, true, j1.CreatedAt.After(aftCreation), j1.CreatedAt.Before(befCreation))
+	tBef := j1.CreatedAt
+
+	befInsertion := time.Now()
 	assert.NoError(t, store.CreateJob(&j1))
+	aftInsertion := time.Now()
+
+	tAft := j1.CreatedAt
+	assert.Equal(t, tBef, tAft)
```

Comment:
note for reviewers: this is an assertion that fails the code before this PR, but something feels this might be a cause for mandlebug?

Reply:
substituted with another assertion that also fails on the old code